lib/engine/game/g_1822/game.rb

```diff
@@ -3451,7 +3451,7 @@ def company_choices_lcdr(company, time)
         end
 
         def company_choices_lur(company, time)
-          return {} if time != :issue || !company.owner&.corporation?
+          return {} if !(time == :token || time == :track || time == :issue) || !company.owner&.corporation?
```

Comment:
would probably split the two sides of the or into two to make it clearer
return {} unless company.owner&.corporation?

Reply:
Sure it makes sense :)